experiment_buddy/experiment_buddy.py

```diff
@@ -66,13 +65,11 @@ def register(config_params):
 
 
 def _is_valid_hyperparam(key, value):
-    if key.startswith("__") and key.endswith("__"):
-        return False
-    if key == "_":
-        return False
-    if isinstance(value, (types.FunctionType, types.MethodType, types.ModuleType)):
-        return False
-    return True
+    return not any([
```

Comment:
does this change add any functionality?
the extended if is longer but more explicit